src/DroneSafetyApi/Data/ExampleHazardRepository.cs

```diff
@@ -9,7 +9,7 @@ namespace DroneSafetyApi.Data
 {
     public class ExampleHazardRepository : IHazardRepository
```

Comment:
Perhaps we can given a short explanation of what the examples repository does, only for the class